Misc/votvlibrary.lua

```diff
@@ -129,9 +130,9 @@ function move_contact_air(inst, angle, amount)
 	return x, y
 end
 
-function move_in_direction(inst, angle, amount)
-	inst.x = inst.x + math.cos(math.rad(angle)) * amount
-	inst.y = inst.y + math.sin(math.rad(angle)) * amount
+function move_in_direction(inst, angle, amount, isDeg)
```

Comment:
comment explaining the function since this is a fairly generic name. What is moving in direction? Actor? Tech's deployables? etc

and a comment explaining why we need the isDeg flag, it isnt obvious why this input isnt standardized among calls